Pipfile

```diff
@@ -1,6 +1,6 @@
 [[source]]
 url = "https://pypi.python.org/simple"
-verify_ssl = true
+verify_ssl = false
```

Comment:
Could you let me know why did you set verify_ssl false here? because it's the official PyPi repository I don't think this is a good idea.

Reply:
Was seeing this error ConnectError: [SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:1108) and couldn't figure out way to turn off strict SSL verification. Since the client was sent as a parameter added verify flag, but support for setting custom **client_options** would help too.

Reply:
Is [this](https://github.com/pypa/pip/issues/4205) the cause of your [SSL: CERTIFICATE_VERIFY_FAILED] problem?

Reply:
Yes, I think I overlooked this and got included in PR. Will exclude this changes in next commit.